reactioncommerce/reaction-admin
# Fix: Product editor does not detect publishable changes sometimes

Signed-off-by: jrw421 <jessica.wolvington@gmail.com>

Resolves #251
Impact: **major**
Type: **bugfix**

## Issue
The product editor sometimes does not detect publishable changes.

## Solution
Sometimes on a form update the product hashes would not update to enable the publish button. By refetching the product on an update of the product or a variant the product hashes are the most up to date and the publish button is correctly enabled.

## Breaking changes
None expected.

## Testing
1. Create a new product and add basic information, including an attribute label to the variant, save and publish.
2. Add price to the variant and save.
3. Edit basic information in the product or variant and save.
4. Observe publish button is now enabled after each save.
